[4.0] Simplified docs (#834)
bigbluebutton/bbb-install: README.md

<img width="1012" alt="bbb-install-banner" src="https://user-images.githubusercontent.com/1273169/141152865-f497c883-6c96-41c6-9040-613b0858878a.png">

# bbb-install

> `bbb-install.sh` is a shell script that installs, upgrades, and configures a BigBlueButton server in under 30 minutes. Use it on a fresh Ubuntu server that meets the [minimum requirements](https://docs.bigbluebutton.org/administration/install#minimum-server-requirements) and has a fully qualified domain name (FQDN) pointing to it.

Author: ```suggestion
`bbb-install.sh` is a shell script that installs, upgrades, and configures a BigBlueButton server in under 20 minutes. Use it on a fresh Ubuntu server that meets the [minimum requirements](https://docs.bigbluebutton.org/administration/install#minimum-server-requirements) and has a fully qualified domain name (FQDN) pointing to it.
```

Author: For me it's been ~10 mins consistently.

A TURN host needs its own FQDN (with A and AAAA records), its own email for Let's Encrypt, and a shared secret (any 8–16 character random string). The installation is two steps: (1) install coturn on a separate host with `-c <fqdn>:<secret> -e <email>` and no `-v`; (2) point BigBlueButton at it by re-running the BBB install with `-c <fqdn>:<secret>` appended. One TURN server can be shared across multiple BigBlueButton servers. See `-h` for the exact commands.

### Storing recordings on a separate volume

`/var/bigbluebutton` can grow large. Pass `-m /mnt/recordings` to symlink it to a separately-mounted volume.

## Upgrading

Re-run the same `bbb-install.sh` command you used to install. The script upgrades BigBlueButton and all installed add-ons (Greenlight, Keycloak, LTI) to the latest stable version of the same version line. To jump to a newer line, change `-v`.

Updates are announced on the [bigbluebutton-dev](https://groups.google.com/forum/#!forum/bigbluebutton-dev) mailing list.

## Troubleshooting

If the script fails or misbehaves, open a [GitHub issue](https://github.com/bigbluebutton/bbb-install/issues) with steps to reproduce.

> For help with BigBlueButton itself, post to the [BigBlueButton Setup](https://bigbluebutton.org/support/community/) community.

Reviewer: this link leads to a 404 error page

Author: ```suggestion
For help with BigBlueButton itself, post to the [BigBlueButton community](https://bigbluebutton.org/community-support/).
```

## Limitations

- If you are behind an external firewall, `bbb-install.sh` won't configure it. See [External firewall](#external-firewall).

> - Cross-major-OS upgrades are not supported. If you're moving from an older Ubuntu release, install on a fresh server and [transfer your recordings](https://docs.bigbluebutton.org/admin/customize.html#transfer-recordings).

Reviewer: this link leads to a 404 error page

Author: ```suggestion
- Cross-major-OS upgrades are not supported. If you're moving from an older Ubuntu release, install on a fresh server and [transfer your recordings](https://docs.bigbluebutton.org/administration/customize/#transfer-recordings).
```
